recipes-core/core-framework/bulkiointerfaces_2.2.2.bb

```diff
@@ -25,6 +25,11 @@ DESCRIPTION = "REDHAWK Framework BulkIO Interfaces"
 DEPENDS += "redhawk"
 RDEPENDS_${PN} = "redhawk"
 RDEPENDS_${PN}-python = "redhawk-python"
+#this was added to fix provide errors
```

Comment:
None of these changes are necessary.  Please remove.

Reply:
I'll do this now.